src/app/lib/utilities/filterPopularStaleData/isDataStale.js

```diff
@@ -0,0 +1,23 @@
+const MAXIMUM_STALE_DATA_TIME = 60 * 60 * 1000; // 60 minutes
+const SHOULD_RENDER_LAST_UPDATED_TIME = 60 * 24 * 60 * 60 * 1000; // 60 days
+
+/**
+ * @param {string} formattedTimestamp A formatted timestamp (e.g. 2019-11-06T16:49:00Z )
+ */
+
+// This is different from the timestamps under the MostReadLinks and is to do with preventing stale data
+// caused by potential failed PopAPI updates.
+export const isDataStale = formattedTimestamp => {
+  // Note in storybook this value will be 'Friday, 9 August 2019 14:04:14 GMT'
+  // Because of timemachine (simorgh/.storybook/time-machine.js)
+  const now = Date.now();
```

Comment:
Tiny observation - could hold now in a const outside this func so it could also be used on ln 20? 🤷‍♂️

Reply:
That can get inaccurate since now will be calculated only once